src/views/Profile/PersonalInfo/PersonalInfo.tsx

```diff
@@ -395,7 +427,7 @@ export const PersonalInfo: React.FC<IEditAuthorProps> = ({
           type="sign"
           label={i18n.t(langTokens.common.acceptChanges)}
           className={classes.BasicButton}
-          onClick={buttonClickHandler}
```

Comment:
why it did not work before? to me it looks legit

Reply:
I think its working, idk why i changed this 🙂